builder/bundles.go

```diff
@@ -480,30 +471,146 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func downloadRpm(packagerCmd []string, rpmList []string, baseDir string, downloadRetries int) (error, string) {
+	// Retry RPM downloads to avoid timeout failures due to slow network
+	_, err := downloadRpms(packagerCmd, rpmList, baseDir, downloadRetries)
+	if err != nil {
+		return err, ""
+	}
+	path := baseDir + rpmDir
+
+	fullRpmPath, err := filepath.Glob(filepath.Join(path, "clear-*"))
```

Comment:
Can't we use `upstreamver` here to form the path instead of using `Glob()` ?

Reply:
I think its done because dnf downloads rpms to directories like
`update/image/10/full/var/cache/yum/clear/clear-1e95707add22ef34`